<launch>
  <!-- Axon Node for ROS 1 -->
  
  <arg name="config_path" default="$(find axon)/config/default_config.yaml" />
  <arg name="dataset_path" default="/data/recordings/dataset.lance" />
  <arg name="auto_start" default="true" />
  
  <node name="axon" pkg="axon" type="axon_node" output="screen">
    <param name="config_path" value="$(arg config_path)" />
    <param name="dataset_path" value="$(arg dataset_path)" />
    <param name="auto_start" value="$(arg auto_start)" />
  </node>
  
</launch>

